internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -189,9 +189,9 @@ type Secret struct {
 }
 
 type Event struct {
```

Comment:
something is not right here
where is the primary key of the table ?
there was a composed primary key of 3 fields if I recall correctly